core/org.osate.ui/plugin.xml

```diff
@@ -525,6 +525,259 @@ censes only apply to the Third Party Software and not any other portion of this
              </adapt>
           </or>
        </definition>
+
+       <!-- open AADL project or any closed project -->
+       <definition id="org.osate.ui.definition.private.isAADLProject">
+          <and>
+             <instanceof value="org.eclipse.core.resources.IProject" />
+             <or>
+                <test
+                    property="org.eclipse.core.resources.projectNature"
+                    value="org.osate.core.aadlnature" />
+                <not>
+                   <test property="org.eclipse.core.resources.open" />
+                </not>
+             </or>
+          </and>
+       </definition>
+
+       <!-- folder in an AADL project -->
+       <definition id="org.osate.ui.definition.private.isAADLFolder">
+          <and>
+             <instanceof value="org.eclipse.core.resources.IFolder" />
+             <test
+                   property="org.eclipse.core.resources.projectNature"
+                   value="org.osate.core.aadlnature" />
+             </and>
+       </definition>
+
+       <!-- Instance model file -->
+       <definition id="org.osate.ui.definition.private.isAAXLFile">
+          <adapt
+                type="org.eclipse.core.resources.IFile">
+             <test
+                   property="org.eclipse.core.resources.contentTypeId"
+                   value="org.osate.core.instanceModelFile">
+             </test>
+          </adapt>
+       </definition>
+
+       <!-- Declarative model file: AADL Package -->
+       <definition id="org.osate.ui.definition.private.isAADLFile">
```

Comment:
better name: isPackageFile
add a definition isPropertySetFile

Reply:
Renamed and added